arangod/Pregel/GraphStore.h

```diff
@@ -130,9 +131,9 @@ class GraphStore final {
   GraphFormat<V, E> const* graphFormat() { return _graphFormat.get(); }
 
   // ====================== NOT THREAD SAFE ===========================
-  void loadShards(WorkerConfig* config,
-                  std::function<void()> const& statusUpdateCallback,
-                  std::function<void()> const& finishedLoadingCallback);
+  auto loadShards(WorkerConfig* config,
+                  std::function<void()> const& statusUpdateCallback)
```

Comment:
TODO: think about how to handle the status updates.

Reply:
Yes, I did not know how to improve that: We want to have status updates during the loading and not only at the end of loading, therefore the worker has to initiate the update, therefore we have to give this callback from the conductor to the worker.